docs/Configuration.md

```diff
@@ -92,7 +92,7 @@ const {defaults} = require('jest-config');
 
 /** @type {import('jest').Config} */
 const config = {
-  moduleFileExtensions: [...defaults.moduleFileExtensions, 'mts', 'cts'],
+  moduleDirectories: [...defaults.moduleDirectories, 'bower_components'],
```

Comment:
why is this changed to `moduleDirectories`?

Reply:
I had the same question. My guess: if `'mts'` and `'cts'` get included in the default, the code in the example does nothing useful for the user. So @yinm was simply trying to find a way to keep the example useful.

Reply:
Yes, @mrazauskas's guess is right. I was trying to find a way to keep the example useful at https://github.com/jestjs/jest/pull/14369/commits/0f3e2615f5b84dd54a290d0be0c6af44eabc321b.